src/Test.Utilities/CSharpSecurityCodeFixVerifier`2+Test.cs

```diff
@@ -17,6 +20,17 @@ public Test()
                 // These analyzers run on generated code by default.
                 TestBehaviors |= TestBehaviors.SkipGeneratedCodeCheck;
             }
+
+            protected override Project ApplyCompilationOptions(Project project)
+            {
+                var newProject = base.ApplyCompilationOptions(project);
+
+                var parseOptions = newProject.ParseOptions.WithFeatures(
```

Comment:
As discussed, I have applied this change for all security tests through this helper.

Reply:
This method override should be moved to the base type `CSharpCodeFixVerifier<TAnalyzer, TCodeFix>.Test`. The feature flag is needed for all tests in 2.9.x branch, not just security tests.

Reply:
Will do.